apps/native-component-list/package.json

```diff
@@ -29,7 +29,7 @@
     "react-navigation-header-buttons": "^1.2.1",
     "react-navigation-material-bottom-tabs": "1.0.0-alpha.4",
     "regl": "^1.3.0",
-    "three": "^0.88.0",
+    "three": "^0.98.0",
```

Comment:
Make sure that this works with android. If I remember correctly there was some arithmetic error.